bin/jsonpedia.sh

```diff
@@ -0,0 +1,8 @@
+/var/lib/spark/bin/spark-submit \
```

Comment:
we also need driver memory here?

Reply:
good to add : 
` set -e ` at the top of this script. so that it will exit if an intermediate step fails